apis/keyvaluestore.proto

```diff
@@ -21,7 +21,7 @@ package keyvaluestore;
 // A simple key-value storage service
 service KeyValueStore {
   // Provides a value for each key request
-  rpc GetValues (stream Request) returns (stream Response) {}
+  rpc GetValues (Request) returns (Response) {}
```

Comment:
This requires the parameter api code and test to be updated as well

Reply:
This change is pushed to main now so you can rebase this PR to pick it up
NB! we do currently have an issue on the unit tests for parameter API on main though, so tests are failing there

Reply:
tests are fixed now